tests/Unit/HttpWorkerTest.php

```diff
@@ -308,6 +308,7 @@ public static function headersDataProvider(): \Traversable
             ['Content-Type' => 'application/x-www-form-urlencoded'],
             ['Content-Type' => new HeaderValue(['value' => ['application/x-www-form-urlencoded']])]
         ];
+        yield [['Content-Type' => new \stdClass()], []];
```

Comment:
I think it doesn't cover the change. It's better to use `(object)['foo' => 'bar']` instead of `stdClass`